README.md

```diff
@@ -16,7 +16,25 @@ Your task is to create a RESTful API for a time clock system.  All required endp
 
 *Timesheet*: A timesheet is a pair of clock in/out timestamps with the number of hours and gross pay calculated.  
 
-*Social Insurance Number (SIN)*: Use this document as a guide for implementing your own validation for SINs. [https://www.straightlineinternational.com/docs/vaildating_canadian_sin.pdf](https://www.straightlineinternational.com/docs/vaildating_canadian_sin.pdf)
+*Social Insurance Number (SIN)*: Use the following method to validate a SIN.
+
+>Let's use this fictitious SIN to demonstrate: `130 692 544`
+>
+>Always multiply the SIN Number by this number: `121 212 121`
+>
+>(Multiply each digit of the top number by each digit of the bottom number)
+>
+>`130 692 544`
+>
+>`121 212 121`
+>
+>`160 394 584`
+>
+>If you get a 2 digit product, add the digits together. Notice here that 6 * 2 = 12, add 1 and 2 together and get 3.
```

Comment:
I think the way this is currently written is a bit confusing.  How about something like this:

Multiple each digit of the SIN by the corresponding digit of the number `121 212 121`.  If you get a 2 digit product, add the digits together.

`130 692 544`

`121 212 121`

`160 394 584`

Notice here that 6 * 2 = 12; add 1 and 2 together and get 3.

Reply:
Done